libecs/include/pivot/ecs/Core/SceneManager.hxx

```diff
@@ -14,8 +14,7 @@ const LevelId MAX_LEVELS = UINT16_MAX;    // 65535
 class SceneManager
 {
 public:
-    /// Init scene list
-    void Init();
+    SceneManager(): _currentActiveLevel(-1){};
```

Comment:
Using -1 might not have the behaviour you want. 
```cpp
uint16_t val = -1; // is equal to INT16_MAX
```
If you want it to be an invalid value, prefer `std::optional` or a separate boolean.